project/src/styl/main.styl

```diff
@@ -81,6 +81,28 @@ main.out
       width 20px
       height 20px
       background url('../../images/game_icon.svg') no-repeat
```

Comment:
That's interesting stuff! But it's not related to friends feature. Can you extract this and make a separate pull request for the icon theme?